spec/serializers/current_user_serializer_spec.rb

```diff
@@ -352,4 +352,18 @@
       expect(initial_count).to eq(final_count)
     end
   end
+
+  describe "#featured_topic" do
+    fab!(:featured_topic) { Fabricate(:topic) }
+
+    before { user.user_profile.update(featured_topic_id: featured_topic.id) }
```

Comment:
```suggestion
    before { user.user_profile.update!(featured_topic_id: featured_topic.id) }
```

Minor one but in tests it is better to raise an error if update fails.

Reply:
Thought I did !